### ISSUE TITLE
Memrise course for mastery

### ISSUE THREAD

**Issue by EsmaeilSaleh** [tridactyl/tridactyl#4231]
I've created this memrise course to memorise the most useful keybindings.

If everything seems under control, you're not going fast enough. Let's review The mighty Tridactyl's main Shortcuts. Replace Firefox's default control mechanism with one modeled on the one true editor, Vim.

[Memrise Course: Tridactyl's main Shortcut](https://app.memrise.com/course/5995499/tridactyls-main-shortcuts/)

**Comment by bovine3dom** [comment]
This looks really cool, thanks! We could add links to it from the tutorial and the readme, is there anywhere else you would think to look for it?

**Comment by EsmaeilSaleh** [comment]
Your welcome, and Thank **YOU** for the amazing tridactyl. Yes, that would do it thanks again.

**Comment by bovine3dom** [comment]
@EsmaeilSaleh the course has disappeared, do you have an updated link? https://app.memrise.com/course/5995499/tridactyls-main-shortcuts/

**Comment by EsmaeilSaleh** [comment]
> @EsmaeilSaleh the course has disappeared, do you have an updated link? https://app.memrise.com/course/5995499/tridactyls-main-shortcuts/


No, It was not me, apparently Memerise guys have changed the main link and added `community` routing to community courses here is the new link:

https://app.memrise.com/community/course/5995499/tridactyls-main-shortcuts/

**Comment by EsmaeilSaleh** [comment]
> @EsmaeilSaleh the course has disappeared, do you have an updated link? https://app.memrise.com/course/5995499/tridactyls-main-shortcuts/


This is Memrise team's answer to my email for the problem:

We're sorry to hear you've encountered this problem due to a recent change on our end. The courses haven't been removed but the URLs have changed. Please allow us to share a bit of context:
 
While the course-creation feature, groups and general community courses are still available on [memrise.com](http://memrise.com/), **it's important to note that we plan to sunset these soon**. We've been supporting these for many years with the understanding of how important they've been for our long-term users. However, we've reached a point where supporting the Classic Memrise is no longer financially or technically viable.
 
As part of the sunsetting, we aim to stop more people signing up to community courses as we want to avoid any further disappointment later down the line. That's why anyone signed out of Memrise can no longer view them, as you've experienced.

If you do want still want to share your course with others, please take these steps:
1. Log in on the website ([memrise.com](http://amemrise.com/)). 
2. Copy the URL to the course you want to share.
3. Ensure those you are sharing with are also logged into Memrise on the website (not app) before clicking on the URL you have shared.
 
We hope you understand.
 
Kind regards,
Memrise team

**Comment by bovine3dom** [comment]
Thanks for investigating. It's a shame that they're getting rid of them :(

**Comment by EsmaeilSaleh** [comment]
Yes, unfortunately. I exported it to an Anki course to save for anyone who might be interested in the future.
Here is it's Anki equivalent: 
https://ankiweb.net/shared/info/1012088889

**Comment by bovine3dom** [comment]
Thanks, I'll reopen this issue to remind me to change the Memrise links to point to it
